x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -256,7 +256,8 @@ URI such as `urn:oid:0.9.2342.19200300.100.1.1`, however there are some
 additional names that can be used:
 
 `nameid`::
-    This uses the SAML `NameID` value instead of a SAML attribute. SAML
+    This uses the SAML `NameID` value (all leading and trailing whitespace removed)
+    instead of a SAML attribute. SAML
```

Comment:
This and the following change for `nameid:persistent` are the only changes of this PR. It unfortunately includes a bunch of whitespace changes which are distracting but can be turned off by selecting "Hide whitespace" in GitHub's Diff view setting.